spec/commands/get_pull_request_reviews_info_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe GetPullRequestReviewsInfo do
+  let(:github_user) { create(:github_user) }
+  let(:monkeyci) { create(:github_user, login: 'monkeyci') }
+  let(:pull_request) { create(:pull_request, owner: github_user) }
+
+  def perform(*_args)
+    described_class.for(*_args)
+  end
+
+  context "without any reviews" do
+    it "return empty" do
+      expect(perform(pull_request: pull_request)).to be_empty
+    end
+  end
+
+  context "with reviews" do
+    let!(:monkey_review) do
+      create(:pull_request_review,
+             github_user: monkeyci,
+             pull_request: pull_request,
+             created_at: Time.zone.now - 5.minutes,
+             approved_at: Time.zone.now + 2.minutes)
+    end
+    let!(:human_review) do
+      create(:pull_request_review,
+             github_user: github_user,
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.